index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,500;1,600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@0,400;1,500&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Portifolio</title>
+  </head>
+  <body>
+    <header>
+      <img src="img/image_geometry_1.jpg" alt="square" class="geometry" />
+      <nav>
+        <img src="img/ic_menu@2x.jpg" alt="menu" class="menu-button" />
+      </nav>
+    </header>
+    <section class="headline">
+      <h1>Meqdam<br />Al-qudah</h1>
+
+      <blockquote>
+        Hello! I am a full-stack web developer! I can help you build a product,
+        feature or website. Taka a look of my works. If you like what you see
+        and have a project you need coded, don’t hesitate and contact me.
+      </blockquote>
+      <button>Start collaboration</button>
+    </section>
```

Comment:
Look at the image below. According to the template:

- [ ] There should a little more top and right margin to your hamburger menu
- [ ] Overall, you should reduce the left and right margins of your headline section
- [ ] Your **Start Collaboration** button should have a similar width as the width of your headline

![meqdam-jpg](https://user-images.githubusercontent.com/36542986/138185403-e30395a5-0b46-4a9f-a5b8-27f68e37a153.PNG)